Cargo.toml

```diff
@@ -38,10 +38,9 @@ license = "MIT"
 repository = "https://github.com/denoland/deno"
 
 [workspace.dependencies]
-v8 = { version = "0.74.1", default-features = false }
```

Comment:
any rationale why this got removed (it actually broke the homebrew build)?

Reply:
It's easier for us to manage version bumps if they flow from the upstream deno_core dep. rusty_v8, deno_core, serde_v8 and deno_ops all need to bump in lockstep.